src/util/options.schema.json

```diff
@@ -1306,6 +1344,18 @@
             },
             "default": []
           }
+        },
+        "array": {
+          "title": "annotation.array",
+          "description": "Enable manual annotation of arrays with desired domain",
+          "type": "boolean",
+          "default": false
+        },
+        "track_apron": {
+          "title": "annotation.track_apron",
```

Comment:
Above attribute annotations have their option names exactly match the attribute names. Would be good for consistency to continue that.